CHANGELOG.next.asciidoc

```diff
@@ -241,6 +241,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Populate more ECS fields in the Suricata module. {pull}10006[10006]
 - Add ISO8601 timestamp support in syslog metricset. {issue}8716[8716] {pull}10736[10736]
 - Add more info to message logged when a duplicated symlink file is found {pull}10845[10845]
+- Add configuration knob for auto-discover hints to control whether log harvesting is enabled for the pod/container. {issue}10811[10811]
```

Comment:
{pull}10911[10911] will be good to add here.

Reply:
Sure, will do before the final merge. This was added before the pull number was generated. Thanks!

Reply:
Please approve if everything is OK. I'd like to get this in ASAP, thanks.